src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -724,7 +724,7 @@ void GNAPlugin::LoadNetwork(const CNNNetwork& _network) {
         manager.register_pass<ngraph::pass::ConvertOpSet3ToOpSet2>();
         manager.register_pass<ngraph::pass::ConvertOpSet2ToOpSet1>();
         manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
-        manager.register_pass<ov::intel_gna::pass::MarkupFusableTranspose>();
+        //manager.register_pass<ov::intel_gna::pass::MarkupFusableTranspose>();
```

Comment:
Why did you comment out it? This will lead to accuracy degradation on old kaldi IRs

Reply:
see "do_not_review" and "WIP" label :)

Reply:
removed